File: con_div.java
import java.util.*;
public class con_div {
 
    public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
        int sum=1;
        int n;
        System.out.print("enter any number till you need even number = ");
        n = sc.nextInt();
            while(n>0){
                if(n%3==0 || n%5==0){
                    n--;
                    continue;
                    
                }
     System.out.println(n);
     n--;
 }

        }
    }

File: con_div.cpp

#include <iostream>
using namespace std;
int main() {
    
 int n;
 cout<<"enter the number  till u need even number = ";
 cin>>n;
 
 while(n>0){
     if(n%3==0 || n%5==0){
         n--;
         continue;
     }
     
     cout<<n<<endl;
     
     n--;
 }
 
    return 0;
}
